src/controller/task.spec.ts

```diff
@@ -224,10 +224,10 @@ describe('Tasks', () => {
         const { response: { status, data } } = err
         expect(status).to.equal(400)
         expect(data.message).to.equal('Validation error')
-        expect(data[0].isString).to.equal('title must be a string')
-        expect(data[1].minLength).to.equal('description must be longer than or equal to 1 characters')
-        expect(data[2].isEnum).to.equal('importance must be a valid enum value')
-        expect(data[3].isEnum).to.equal('status must be a valid enum value')
+        expect(data.validationErrors).to.contain('title must be a string')
```

Comment:
Instead of multiple expect, you could write one with `expect(data.validationErrors).to.be.an('array').deep.equal([list of error strings])`

Reply:
fixed 646a1d154f5832f4e0f0b11a67dc4f31d216413b